packages/utils/test/testUtils/getTestValidator.ts

```diff
@@ -0,0 +1,48 @@
+import isEmpty from 'lodash/isEmpty';
+
+import { ValidationData, ValidatorType } from '../../src';
+
+export interface TestValidatorParams {
+  isValid?: boolean[];
+  data?: ValidationData[];
+}
+
+export interface TestValidatorType extends ValidatorType {
+  // eslint-disable-next-line no-unused-vars
+  setReturnValues(params?: TestValidatorParams): void;
```

Comment:
Feeling conflicted about this because mocking the validator seems like the correct thing to do, but I think using an actual validator gives us much more confidence in our test suite.

Reply:
I know the feeling, but the point of abstracting the utils and validator was to break a circular dependency between the two features. One thing we can consider doing in another pass after I complete the ajv v6 package is to figure out a way to run all the schema utils tests in the ajv 6 validator world using the actual validator. rather than the mock... I would effectively make the `setReturnValues()` function in ajv v6 to be a no-op.